api/apps/geoprocessing/test/integration/marxan-run/marxan-run.e2e-spec.ts

```diff
@@ -1,28 +1,35 @@
-import { INestApplication, Injectable } from '@nestjs/common';
+import { INestApplication } from '@nestjs/common';
 import { Test, TestingModule } from '@nestjs/testing';
-import { copySync } from 'fs-extra';
+import AxiosMockAdapter from 'axios-mock-adapter';
```

Comment:
I'd stick to use nock, it's library agnostic and a bit easier to use

Reply:
https://github.com/Vizzuality/marxan-cloud/pull/293/commits/1d75fdcb196091abd991263952816edf49b0d7f2